src/components/MediaPlayer/MediaPlayerComponent.js

```diff
@@ -0,0 +1,69 @@
+import { ProgressBarComponent } from "./ProgressBar/ProgressBarComponent";
+import { AudioInfoComponent } from "./AudioInfo/AudioInfoComponent";
+import { RatingComponent } from "./Rating/RatingComponent";
+import { MainControlComponent } from "./MainControl/MainControlComponent";
+import { DotsMenuComponent } from "../DotsMenu/DotsMenu";
+
+import playerTemplate from "./MediaPlayerComponent.html";
+
+export class MediaPlayerComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  querySelectors() {
+    this.buttons = this.mountPoint.querySelector(".media-player__buttons");
+    this.progressBar = this.mountPoint.querySelector(
+      ".media-player__progress-bar"
+    );
+    this.mainControl = this.mountPoint.querySelector(
+      ".media-player__main-control"
+    );
+    this.volumeBar = this.mountPoint.querySelector(".media-player__volume-bar");
+    this.audioInfo = this.mountPoint.querySelector(".media-player__audio-info");
+    this.audioRating = this.mountPoint.querySelector(
+      ".media-player__audio-rating"
+    );
+    this.dotsMenuPoint = this.mountPoint.querySelector(
+      ".media-player__dots-menu"
+    );
+  }
+
+  mountChildren() {
+    this.mainControlPannel = new MainControlComponent(this.mainControl, {
+      song:
+        "http://drivemusic.me/dl/ar8_BdKPhBpvPyoPFMdryQ/1540528751/download_music/2013/06/jazzamor-way-back.mp3"
+    });
+    this.mainControlPannel.mount();
+    this.audioProgressBar = new ProgressBarComponent(this.progressBar);
+    this.audioProgressBar.mount();
+    this.audioInfoComponent = new AudioInfoComponent(this.audioInfo, {
```

Comment:
please declare a constant in the top level of this file (outside class) - and move it there